web_src/js/components/RepoContributors.vue

```diff
@@ -36,6 +36,18 @@ for (const [key, value] of Object.entries(colors)) {
   colors[key] = getColor(value);
 }
 
+const customEventListener = {
+  id: 'customEventListener',
+  afterEvent: (chart, args, opts) => {
+    // event will be replayed from chart.update when reset zoom,
+    // so we need to check whether args.replay is true to avoid call loops
+    if (args.event.type === 'dblclick' && opts.chartType === 'main' && args.replay !== true) {
```

Comment:
```suggestion
    if (args.event.type === 'dblclick' && opts.chartType === 'main' && !args.replay) {
```

I can't edit the pr somehow. Can you apply this change?

Reply:
OK, I could add a commit from github ui.